src/main/java/io/cdap/plugin/ariba/source/connector/AribaConnector.java

```diff
@@ -64,17 +77,23 @@
 @Plugin(type = Connector.PLUGIN_TYPE)
 @Name(ResourceConstants.PLUGIN_NAME)
 @Description("Connection to access data from Ariba.")
-public class AribaConnector implements Connector {
+public class AribaConnector implements DirectConnector {
   private static final Logger LOG = LoggerFactory.getLogger(AribaConnector.class);
   private static final String METADATA_PATH = "api/analytics-reporting-view/v1";
   private static final String ENTITY_TYPE_TEMPLATE = "template";
   private static final String VIEW_TEMPLATE = "viewTemplateName";
   private static final String PATH_SEGMENT = "%s/viewTemplates";
+  private static final String DATA_PATH_SEGMENT = "%s/views";
   private static final String RECORDS = "Records";
   private static final String TOKEN = "PageToken";
+  private static final String API_KEY = "apiKey";
+  private static final String TEMPLATE_NAME = "templateName";
```

Comment:
All the variables which are getting used in other classes also, move them to single class and use.

Reply:
done